src/resolvers/ResolverBuilder.ts

```diff
@@ -49,12 +50,13 @@ export class ResolverBuilder {
     return {
       fieldName: fieldName,
```

Comment:
Do we need all those fields. If we define implementation to contain name as well we might not need it as separate element